chia/pools/pool_wallet.py

```diff
@@ -70,6 +69,7 @@ class PoolWallet:
     standard_wallet: Wallet
     wallet_id: int
     singleton_list: List[Coin]
+    _pool_wallet_index_cache: Optional[uint32]
```

Comment:
```suggestion
    _owner_sk_and_index: Optional[Tuple[uint32, PrivateKey]]
```

Wouldn't something like above (or just have two members) make sense to avoid lookups for each signature?

Reply:
yeah that's a good change thanks